requirements.txt

```diff
@@ -1,11 +1,7 @@
 praw==4.4.0
 redis<3.0.0
-hiredis
-better_exceptions
```

Comment:
Nuking it from the dependencies doesn't remove it from being installed if don manually. How are you using it? Is that on the dev side or when debugging in production?